quipucords/scanner/network/inspect_callback.py

```diff
@@ -131,7 +131,7 @@ def handle_result(self, result):
                 self._finalize_host(host, SystemInspectionResult.SUCCESS)
             else:
                 processed_value = process.process(
-                    self.scan_task, host_facts, key, value, host)
+                    self.scan_task, self._ansible_facts, key, value, host)
```

Comment:
I think you want `self._ansible_facts[host]`. `self.ansible_facts` is a dict of dicts, but `self._ansible_facts[host]` will be a dict of facts.

Reply:
I understand your point. The issue with this is that when result handling starts for each host, the host is not yet in the `self._ansible_facts` dictionary. Only after a result has been processed will the host be added to the dictionary which is why I just sent the `self._ansible_facts` and then grab the facts for the host inside of process if the dictionary isn't empty.